actions/image_partition_action.go

```diff
@@ -95,6 +96,9 @@ for partition.
 - flags -- list of additional flags for partition compatible with parted(8)
 'set' command.
 
+- partattrs -- list of GPT partition attribute bits. See description for
```

Comment:
Can you actually add the usage notes (and some examples?) here, rather than pointing to sfdisk manpage ?

I'd like to understand if adding a new parameter is the right way to go.

Reply:
@obbardc Added now. Let me know what you think.